src/main/java/dev/jbang/cli/Info.java

```diff
@@ -308,3 +317,42 @@ public Integer doCall() throws IOException {
 		return EXIT_OK;
 	}
 }
+
+@CommandLine.Command(name = "docs", description = "Open the documentation file into the default browser.")
+class Docs extends BaseInfoCommand {
+
+	@Override
+	public Integer doCall() throws IOException {
+		ScriptInfo info = getInfo(false);
+		System.out.println("Command invoked: " + info.docs);
+
+		URI uri = validateDocsReferenceAndTransformToUri(info);
+		Desktop.getDesktop().browse(uri);
+		return EXIT_OK;
+	}
+
+	private static URI validateDocsReferenceAndTransformToUri(ScriptInfo info) {
+		if (info.docs.startsWith("http")) {
```

Comment:
Once the type of info.docs is `ResourceRef` none if this is necessary because that type handles all of the URL vs absolute vs relative paths etc.